src/main/java/io/debezium/connector/db2/Db2ConnectorConfig.java

```diff
@@ -226,62 +227,36 @@ public static SnapshotIsolationMode parse(String value, String defaultValue) {
                     + "In '" + SnapshotIsolationMode.READ_UNCOMMITTED.getValue()
                     + "' mode neither table nor row-level locks are acquired, but connector does not guarantee snapshot consistency.");
 
+    private static final ConfigDefinition CONFIG_DEFINITION = HistorizedRelationalDatabaseConnectorConfig.CONFIG_DEFINITION.edit()
+            .name("Db2")
+            .type(
+                    HOSTNAME,
+                    PORT,
+                    USER,
+                    PASSWORD,
+                    DATABASE_NAME)
+            .connector(
+                    SNAPSHOT_MODE,
+                    BINARY_HANDLING_MODE,
+                    INCREMENTAL_SNAPSHOT_CHUNK_SIZE,
+                    INCREMENTAL_SNAPSHOT_ALLOW_SCHEMA_CHANGES)
+            .excluding(
+                    TABLE_BLACKLIST,
+                    TABLE_EXCLUDE_LIST,
```

Comment:
@ani-sha Is it necessary to exluse table and column exclude/include? IMHO ther are supported, aren't they? OTH {{SCHEMA}} params are not supported so same as for SQL Server.

Reply:
Yes right, I had assumed the opposite. I have updated them now.